gobblin-config-management/gobblin-config-core/src/main/java/gobblin/config/store/api/VersionDoesNotExistException.java

```diff
@@ -0,0 +1,17 @@
+package gobblin.config.store.api;
+
+public class VersionDoesNotExistException extends Exception {
+
+  /**
+   * 
+   */
+  private static final long serialVersionUID = 2736458021800944664L;
+
+  public VersionDoesNotExistException(String message) {
```

Comment:
I think this exception should take a store URI and a version instead of a generic message. It should also define getters for those fields.